js/app/modules/contentGroup/contentGroup.js

```diff
@@ -115,7 +139,23 @@ const ContentGroup = new Module.Class({
             no_show_all: true,
             vexpand: true,
         });
-        this._stack.add_named(this._arrangement, CONTENT_PAGE_NAME);
+
+        let see_more_button = new Gtk.Button({
+            halign: Gtk.Align.CENTER,
+            label: _('See more'),
+            no_show_all: true,
+            visible: this.paginate,
+        });
+
+        see_more_button.get_style_context().add_class(Utils.get_element_style_class(ContentGroup, 'paginate'));
+        let content_grid = new Gtk.Grid({
+            orientation: Gtk.Orientation.VERTICAL,
+            visible: true,
+        });
+        content_grid.add(this._arrangement);
+        content_grid.add(see_more_button);
```

Comment:
I'd suggest to avoid creating and adding the button if `this.paginate` was false, since we don't ever plan to support the property being non-construct-only.